src/pages/patientView/genomicOverview/Tracks.tsx

```diff
@@ -31,7 +31,8 @@ export default class Tracks extends React.Component<TracksPropTypes, {}> {
         // --- end of raphael config ---
 
         // --- chromosome chart ---
-        var chmInfo = tracksHelper.getChmInfo();
+        const chmInfo = tracksHelper.getChmInfo( this.props.mutations?
+            this.props.mutations[0]!.ncbiBuild: "GRCh37");
```

Comment:
if that is a default chromosome, you should make it a constant.  e.g. DEFAULT_CHROMOSOME at top of file

Reply:
agree. Will make **GRCh37** DEFAULT_GENOME_BUILD for now.

Reply:
made GRCh37 a const.